operations/network/network_management_install_guide/aruba/routed_interface.md

```diff
@@ -0,0 +1,30 @@
+# Routed Interfaces
+
+For platforms: 8400 and 8320 by default all interfaces are configured as routed interfaces with support for both IPv4 and IPv6. 
```

Comment:
what about 8325 and 8360?

Reply:
it would be the same across 84/8300. I will reword.

Reply:
resolved.